inscriptions/ISic001305.xml

```diff
@@ -191,17 +196,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140797">
-                <ab>
-                    <lb n="1"/>
-
-
-                    Λούκιε ❦<lb n="2"/>Καίλιε ❦<lb n="3"/>Φῆλιξ, ❦<lb n="4"/>χρηστὲ<lb n="5"/> χαῖρε. ❦
-                    <lb n="6"/>
-                </ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
+            <ab>
+                <lb n="1"/><persName type="attested"><name type="praenomen">Λούκιε</name> <g type="hedera"/>
```

Comment:
the hedera and similar symbols, like interpuncts, are done using the examples in the character declaration file at https://github.com/ISicily/ISicily/blob/master/alists/charDecl.xml (as per 'glyphs and symbols' in https://github.com/ISicily/ISicily/wiki/Encoding:-text, using ref to point to the example, and encoding the actual unicode symbol whenever it exists, so:
`<g ref="#ivy-leaf">❦</g>`